magento/magento2
# Resolved #18025 - Rounding problem on prices when adding tax to display prices

### Description

Resolved rounding problem on prices when adding tax to display prices.
### Fixed Issues (if relevant)

1. magento/magento2#18025: Rounding problem on prices when adding tax to display prices

### Manual testing scenarios

1.  Set _Configuration > Tax > Calculation Settings_ to following:
![tax_calculation_settings](https://user-images.githubusercontent.com/5977769/45406384-682dde00-b677-11e8-85ec-e1400d2bc6ec.png)
2. Set  _Configuration > Tax > Price Display Settings_ to _Including Tax_
3. Tax should be added to price set in product data when getting amount from classes extending Magento\Framework\Pricing\Price\AbstractPrice. Example Affirm Water Bottle with price of 7.00 with tax of 25 percent results final value of 8.75

### Contribution checklist
